cmake/CheckDGtalDependencies.cmake

```diff
@@ -14,9 +14,17 @@ set(Boost_USE_STATIC_RUNTIME OFF)
 set(Boost_FOUND FALSE)
 FIND_PACKAGE(Boost 1.46.0 REQUIRED)
 if ( Boost_FOUND )
-  message(STATUS "Boost found.")
   include_directories( ${Boost_INCLUDE_DIRS} )
   SET(DGtalLibInc ${DGtalLibInc} ${Boost_INCLUDE_DIRS})
+
+  ## Checking boost/random ( <1.47)
+  STRING (COMPARE LESS "${Boost_MINOR_VERSION}" 47 BOOST_RANDOM_OLD)
```

Comment:
I tried something based on Boost_VERSION but it didn't work (not sure how cmake really compares strings with numerals). I'll try your line, thx